activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ModelControllerIT.java

```diff
@@ -523,7 +523,7 @@ public void should_returnStatusNoContent_when_validatingProcessModelWithValidCon
 
         final ResultActions resultActions = mockMvc
             .perform(multipart("/v1/models/{model_id}/validate", processModel.getId()).file(file))
-            .andExpect(status().isNoContent());
+            .andExpect(status().isBadRequest());
```

Comment:
This is the same situation as with the other test case with changed expectation. Please see the comment there.